packages/containers/src/cmfModule.js

```diff
@@ -0,0 +1,16 @@
+import { cmfConnect } from '@talend/react-cmf';
+import omit from 'lodash/omit';
+import * as allComponents from '@talend/react-components';
+import * as containers from './index';
+
+const components = Object.keys(allComponents).reduce((acc, key) => {
+	if (!acc[key] && typeof allComponents[key] === 'function') {
+		// eslint-disable-next-line no-param-reassign
+		acc[key] = cmfConnect({})(allComponents[key]);
```

Comment:
This doesn't work for class components.
And why is this duplicated in `index.js` and here `cmfMudle.js`